src/core/css-bundler.ts

```diff
@@ -111,8 +111,8 @@ const browserlist = (
     )
   )
 
-const getDraftSpecs = (isDraftSyntaxDisabled: boolean): Drafts =>
-  isDraftSyntaxDisabled
+const getDraftSpecs = (isDraftSyntaxIncluded: boolean): Drafts =>
+  isDraftSyntaxIncluded
```

Comment:
Here, we can use a similar namespace:

```suggestion
  isDraftSpecsIncluded
```

Thoughts?

Reply:
Let us use the naming used by Lightning CSS, `Draft Syntax`, and update the property too. WDYT? @giotramu

Reply:
Sticking to the _Specs_ namespace is better, as this is the [official terminology](https://www.w3.org/Style/CSS/specs.en.html) used by the W3C.

So I suggest `isDraftSpecsIncluded` and `includeDraftSpecs`.